### PR TITLE
Egifter stable

### PR DESCRIPTION
**Pull request by rybons**

### CODE REVIEW

File: terraform/aws/inputs.tf [hunk 1]
```diff
@@ -7,18 +7,19 @@ variable "aws_region" {
 variable "aws_profile" {
   type        = "string"
   description = "The profile on disk. Typically stored in your ~/.aws/config file."
-  default     = "bluepipeline"
+  default     = "egifter"
```
Comment (skylerto) [inline terraform/aws/inputs.tf]: @rybons can we get this as a empty string?

File: terraform/aws/inputs.tf [hunk 2]
```diff
@@ -7,18 +7,19 @@ variable "aws_region" {
...
   type        = "string"
   description = "The path on disk to your private key. This key will be used to establish connectivity to resources."
+  
 }
 
 variable "aws_key_pair_name" {
   type        = "string"
   description = "The name in AWS associated with the public key on your AWS account. Used to allow access to your private key file by placing your public key on resources."
-  default     = "bluepipeline"
+  default     = "egifter"
```
Comment (skylerto) [inline terraform/aws/inputs.tf]: @rybons can we get this as a empty string?

File: terraform/aws/main.tf [hunk 3]
```diff
@@ -52,29 +52,29 @@ resource "aws_security_group" "default" {
...
+// }
 
 resource "aws_instance" "builder" {
   connection {
     user        = "${var.aws_image_user}"
     private_key = "${file("${var.aws_key_pair_file}")}"
   }
 
-  ami                         = "${data.aws_ami.centos.id}"
+  ami                         = "ami-d7e1d2bd"
```
Comment (skylerto) [inline terraform/aws/main.tf]: @rybons can we get this as a variable?

Comment (skylerto) [inline terraform/aws/main.tf]: @rybons can we get this as a variable?

File: terraform/aws/main.tf [hunk 4]
```diff
@@ -52,29 +52,29 @@ resource "aws_security_group" "default" {
 ////////////////////////////////
 // On Prem Builder - Single Instance
 
-data "aws_ami" "centos" {
-  most_recent = true
+// data "aws_ami" "centos" {
```
Comment (skylerto) [inline terraform/aws/main.tf]: @rybons this code can probably be deleted too.

File: terraform/aws/main.tf (hunk 3 above)
Comment (skylerto) [reply terraform/aws/main.tf]: I see the variable, just need to source it here: https://github.com/Indellient/on-prem-builder/pull/6/files#diff-576b2088b5109c94c2eaa89f8724a611R95

File: terraform/aws/main.tf (hunk 4 above)
Comment (rybons) [reply terraform/aws/main.tf]: done

**Comment by gscho** [comment]
@gurkamalhans 
If you can remove the commented out code here I'll approve this also.

**Pull request merged**